pepper-apis/src/main/java/org/broadinstitute/ddp/route/AddProfileRoute.java

```diff
@@ -32,19 +33,14 @@ public Object handle(Request request, Response response, Profile profile) throws
         String userGuid = request.params(RouteConstants.PathParam.USER_GUID);
         LOG.info("Creating profile for user with guid {}", userGuid);
 
-        if (profile == null) {
-            ResponseUtil.halt400ErrorResponse(response, ErrorCodes.MISSING_BODY);
```

Comment:
Can you take a quick look at the API docs and clean those up as necessary?

Reply:
cleaned up API docs.